.circleci/config.yml

```diff
@@ -39,6 +39,7 @@ jobs:
           command: |
             export USER=$(whoami)
             make ubuntu16
+          no_output_timeout: 1h
```

Comment:
Why is it taking more than 10 minutes to build an AMI?  1 hour seems an excessive timeout value to build a base image.